src/VirtoCommerce.CatalogCsvImportModule.Core/Model/CsvProductMappingConfiguration.cs

```diff
@@ -15,15 +15,15 @@ public CsvProductMappingConfiguration()
         public string Delimiter { get; set; }
         public string[] CsvColumns { get; set; }
         public ICollection<CsvProductPropertyMap> PropertyMaps { get; set; }
-        public string[] PropertyCsvColumns { get; set; }
+        public string[] PropertyCsvColumns { get; set; } = new string[0];
 
         public static CsvProductMappingConfiguration GetDefaultConfiguration()
         {
             var retVal = new CsvProductMappingConfiguration { Delimiter = ";" };
 
             var requiredFields = new List<string>();
             var optionalFields = ReflectionUtility.GetPropertyNames<CsvProduct>(x => x.Name, x => x.Id, x => x.Sku, x => x.CategoryPath, x => x.CategoryId, x => x.MainProductId, x => x.PrimaryImage, x => x.AltImage, x => x.SeoUrl, x => x.SeoTitle,
-                                                                                x => x.SeoDescription, x => x.SeoLanguage, x => x.SeoStore, x => x.SeoMetaKeywords, x => x.SeoImageAlternativeText, x => x.Review, x => x.ReviewType, x => x.IsActive, x => x.IsBuyable, x => x.TrackInventory,
```

Comment:
why delete it?

Reply:
I don't know. Looks like some environmental bug or I keystroked something occasionally.
Just turned it back.